dspback/routers/earthchem.py

```diff
@@ -26,9 +37,10 @@ class EarthChemMetadataRoutes(MetadataRoutes):
     @router.post('/metadata/earthchem', tags=["EarthChem"])
     async def create_metadata_repository(self, request: Request, metadata: request_model) -> response_model:
```

Comment:
This was not necessary, but cleans things up.  Before, the ecl resource was created and then followed up with an update with all the metadata.  I updated the create metadata to include all the metadata so we don't have to call an update immediately after a create.